CalculatorAppJMS/CalculatorAppJMS/Program.cs

```diff
@@ -0,0 +1,121 @@
+﻿// See https://aka.ms/new-console-template for more information
+using CalculatorLibrary;
+
+bool endApp = false;
+Calculator calculator = new Calculator();
+int timesCalculatorUsed = 0;
+string userOption;
+bool gag = false;

```

Comment:
what does gag stand for? Could you please rename this bool with a more significant name so I know what it does? 😊

Reply:
Hi, i changed the name for something more explanatory. Its just a bool to set up a silly joke that plays out if you try to view how many times you have you used the calculator after you perform 2 operations. :)